defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-haproxy_exporter_bin_dir: "/usr/local/bin"
+haproxy_exporter_bin_dir: /opt/haproxy-exporter
```

Comment:
I know this is WIP, but any reason we're deviating from where most exporters are installed?

```
$ which node_exporter
/usr/local/bin/node_exporter
$ which mysqld_exporter
/usr/local/bin/mysqld_exporter
```

Reply:
No reason, changed it to match with others.